common/common_test.go

```diff
@@ -20,24 +20,28 @@ import (
 	"path/filepath"
 	"testing"
 
-	cmp "github.com/google/go-cmp/cmp"
 	slsa "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v0.2"
 	"github.com/project-oak/transparent-release/internal/testutil"
 	"github.com/project-oak/transparent-release/pkg/amber"
 )
 
-const testdataPath = "../testdata/"
-const provenanceExamplePath = "schema/amber-slsa-buildtype/v1/example.json"
+const (
+	testdataPath             = "../testdata/"
+	provenanceExamplePath    = "schema/amber-slsa-buildtype/v1/example.json"
+	wantTOMLHash             = "322527c0260e25f0e9a2595bd0d71a52294fe2397a7af76165190fd98de8920d"
+	wantBuilderImageID       = "6e5beabe4ace0e3aaa01ce497f5f1ef30fed7c18c596f35621751176b1ab583d"
+	wantSHA1HexDigitLength   = 40
+	wantSHA256HexDigitLength = 64
+)
 
 func TestComputeBinarySha256Hash(t *testing.T) {
-	want := "3dbf6017c84f2a6be8d1d914ff6da2b9a34829b1846a342b8b73856fa53d4d6b"
-	path := filepath.Join(testdataPath, "build.toml")
+	path := filepath.Join(testdataPath, "static.txt")
 	got, err := computeSha256Hash(path)
 	if err != nil {
 		t.Fatalf("couldn't get SHA256 hash: %v", err)
 	}
-	if got != want {
-		t.Errorf("invalid commit hash: got %s, want %s", got, want)
+	if got != wantTOMLHash {
+		t.Errorf("invalid commit hash: got %s, want %s", got, wantTOMLHash)
```

Comment:
I'm still not sure why this is a commit hash though.

Reply:
Ah! Good catch! I had not noticed it!